django_refugeerights/subscription/tests.py

```diff
@@ -85,6 +85,24 @@ def test_create_subscription(self):
         self.assertEqual(d.lang, "en_GB")
         self.assertEqual(d.schedule.id, self.schedule.id)
 
+    def test_switch_subscription(self):
+        response = self.client.get('/subscription/switch_subscription/',
+                                   {"to_addr": "+278888"},
```

Comment:
You want to pass in the `contact_key` and `messageset_id` here (you should know what parameters your own API accepts!). :)